src/event/event_queue.rs

```diff
@@ -138,19 +138,19 @@ impl EventQueue {
     }
 }
 
-/// Error indicating that a closure was unable to be scheduled to execute on the event queue
-pub struct EventQueueError;
+/// Error indicating that a closure was unable to be scheduled to execute on the event loop.
+pub struct SendError;
```

Comment:
I chose `SendError` but we could also potentially choose `TrySendError`? The `mpsc` module has both types where we only have one. I thought `TrySendError` seemed unnecessarily pedantic, but I'm open to alternatives.

Reply:
`SendError` looks like the correct one. `TrySendError` is for a non-blocking send on a bounded queue. Currently, we only have an unbounded version of `EventQueue`.